packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -265,6 +265,9 @@ export class OdspDocumentService implements IDocumentService {
                 return connection;
             } catch (error) {
                 this.cache.sessionJoinCache.remove(this.joinSessionKey);
+                if (error) {
```

Comment:
if (typeof error === "object" && error !== null)

Reply:
Yes, I was adding this only. Thanks.